CHANGES.md

```diff
@@ -46,6 +46,7 @@ Apollo 1.9.0
 * [feat(apollo-client): add method interestedChangedKeys to ConfigChangeEvent](https://github.com/ctripcorp/apollo/pull/3666)
 * [add More... link for known users](https://github.com/ctripcorp/apollo/pull/3757)
 * [update OIDC documentation](https://github.com/ctripcorp/apollo/pull/3766)
+* [Use queue#take instead of poll to avoid cs](https://github.com/ctripcorp/apollo/pull/3765)
```

Comment:
I noticed you changed the title, we need to also update the description in the changes.md.
btw, does cs mean `context switch`?
```suggestion
* [Use queue#take instead of poll](https://github.com/ctripcorp/apollo/pull/3765)
```

Reply:
Yes, so I think it's no need to poll and then sleep.